tst/testinstall/list.tst

```diff
@@ -0,0 +1,204 @@
+gap> START_TEST("list.tst");
+
+# EQ: for two small lists, 1
+gap> [1] = [2];
+false
+gap> [1] = [1];
+true
+gap> [2, 1] = [2];
+false
+gap> [4] = [4, 5];
+false
+
+# EQ: for two lists, the first or second being empty
+gap> newtype := NewType(ListsFamily, IsList and IsComponentObjectRep
+>                                    and IsAttributeStoringRep);;
```

Comment:
Why do you define a new type for this? Perhaps to bypass some fastpath in the interpreter? Whatever it is, a comment should explain it.